src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -138,4 +140,24 @@ public static Note parseNote(String note) throws ParseException {
         }
         return new Note(trimmedNote);
     }
+
+    /**
+     * Parses a {@code String fileType} into a valid {@code FileType}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code fileType} is invalid.
+     */
+    public static FileType parseFileType(String fileType) throws ParseException {
+        requireNonNull(fileType);
+        String trimmedFileType = fileType.trim().toUpperCase();
+
+        try {
+            FileType type = FileType.valueOf(trimmedFileType);
+            return type;
+        } catch (IllegalArgumentException argex) {
+            throw new ParseException(MESSAGE_CONSTRAINTS);
+        } catch (NullPointerException nullex) {
+            throw new ParseException(MESSAGE_CONSTRAINTS);
+        }
```

Comment:
`NullPointerException` is thrown only when `trimmedFileType` or `FileType` are null. Which in this case, is impossible:
- `FileType` is a class which can never be null
- `trimmedFileType` is created from `fileType`. Since you are able to call a method (e.g. `.trim()`) on `fileType`, this means that `fileType` is not null.

In addition, we can see that the method assumes that `fileType` is not null because of the assertion.